packages/roosterjs-react-command-bar/lib/utils/OutOfBoxCommandBarButtons.tsx

```diff
@@ -277,3 +298,44 @@ function _handleChangeForHighlight(editor: Editor): void {
     const { color } = this.data as ColorInfo;
     setBackgroundColor(editor, color);
 }
+
+function _getIconOnRenderDelegate(asset: { name: string; className?: string }, secondAsset: { name?: string }, highContrastAssetName: string = null): ButtonOnRenderDelegate {
```

Comment:
Do you see us using more than 2 assets at a time? Should we pass a list of assets and then loop over those sequentially? Or do u think it is overkill?

Reply:
We likely won't for more than two. I had thought about it too, so let's see...